### ISSUE TITLE
Telegram comments block is rendered every time when anchor link is clicked

### ISSUE THREAD

**Issue by difhel** [difhel/blog#1]
Open a blog (https://difhel.dev/blog/cython-fastapi-benchmark) and click several times on links in the bottom navigation menu.
![image](https://github.com/difhel/blog/assets/78644136/1f78efb8-683c-4779-8cd9-633a0ee1db80)

**Comment by difhel** [comment]
Can be easily fixed by adding additional check in useEffect:
```ts
const [isDOMRendered, updateDOMRendered] = useState(false);
useEffect(() => {
    if (isDOMRendered) return;
    // ...
    // render Telegram comments
    updateDOMRendered(true);
}, []);
```

**Comment by difhel** [comment]
Fixed in 0ad7f2c
